azure-pipelines-template.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           npm -v
         displayName: "Print version"
       - script: |
-          npm ci
```

Comment:
npm ci is probably better, runs fresh install with lockfile

Reply:
That's true, I've reverted this change because I'm trying to understand what broke Azure CI. But the problem remains, so I'll try something else :)

Reply:
- "chore(ci): add stable and improve azure config" ([6ca6235](https://github.com/webpack/webpack-cli/pull/872/commits/6ca6235fe8018936e0b72d99a64f3736881d0bd5))([azure](https://dev.azure.com/webpack/webpack/_build/results?buildId=2380))
- "chore(azure): change to npm ci" ([8f02bb3](https://github.com/webpack/webpack-cli/pull/872/commits/8f02bb3d0cd2a7f6767afac4748c1c5bc0ba218b))([azure](https://dev.azure.com/webpack/webpack/_build/results?buildId=2426))

Seems that using `npm ci` instead of `npm install` breaks only on "macOS node-6"
It fails with `sh: rimraf: command not found` while executing `npm run bootstrap`

I'm keeping the old `npm install`. Any idea on how to fix this is welcome :)

Reply:
Better always update npm to latest before use npm commands, npm is very buggy

Reply:
You are right, on Mac OS it doesn't update npm to the latest version.
But the fix suggested [here](https://github.com/webpack/webpack-cli/commit/c9c3fea8fc32286e27492605b5c8ec37fe12cecc#r33311253) doesn't work on [Linux](https://dev.azure.com/webpack/webpack/_build/results?buildId=2409)

I'm going to investigate in the next days :)